vite/src/App.tsx

```diff
@@ -0,0 +1,112 @@
+                  <div>
+                    <div className="text-sm">{r.timestamp}</div>
+                    <div className="text-xs">{r.id}</div>
+                    <div className="text-xs text-dailyGray400">Room: {r.room_name}</div>
+                  </div>
+                  <div className="flex gap-3">
+                    <button onClick={() => processRecording('audio_enhance', r.id)} className={`px-3 py-1.5 bg-dailyDarkSky hover:bg-dailyDarkSkyHover rounded text-white border-solid text-sm`}>Enhance audio</button>
+                    <button onClick={() => processRecording('text_to_video_lipsync', r.id)} className={`px-3 py-1.5 bg-dailyDarkOlive hover:bg-dailyDarkOliveHover rounded text-white border-solid text-sm`}>TTS Lipsync</button>
+                    <button onClick={() => processRecording('video_dubbing', r.id)} className={`px-3 py-1.5 bg-dailyOrange hover:bg-dailyOrangeHover rounded text-white border-solid text-sm`}>Translate & Dub</button>
+                  </div>
+                </li>
+              ))}
+            </ul>
+          </aside>
+          <ul className="grid grid-cols-2 gap-3 w-full col-span-2">
+            {processedVideo &&
+            <>
+            <div className="text-center text-gray-500">
+              <video controls width="750" height="500" src={`${processedVideo?.original_video}`}/>
+              <p className="mt-4">Original video</p>
+            </div>
+            <div className="text-center text-white">
+              <video controls width="750" height="500" src={`/${processedVideo?.processed_video}`}/>
+              <p className="mt-4">Processed video</p>
+            </div>
+            </>
+            }
+          </ul>
+        </div>
+      </div>
+    </div>
+    )
+  }
+
+  const [recordings, setRecordings] = useState<DailyRecording[]>()
+  const [processedVideo, setProcessedVideo] = useState<ProcessedVideo>()
+
+  function getRecordings() {
+    fetch(`${apiURL}/recordings`, { method: 'GET' })
+      .then((res) => {
+        if (res.ok === false) {
+          throw Error(`upload request failed: ${res.status}`);
+        }
+        return res.json();
+      })
+      .then((data) => {
+        const { recordings } = data;
+        if (recordings) {
+          setRecordings(recordings);
+        }
+      })
+      .catch((e) => {
+        console.error('Failed to process uploaded video:', e);
+      });
+  }
+
+  function processRecording(endpoint: string, recording_id: string) {
+    fetch(`${apiURL}/${endpoint}/${recording_id}`, { method: 'POST' })
+    .then((res) => {
+      if (res.ok === false) {
+        throw Error(`upload request failed: ${res.status}`);
+      }
+      return res.json();
+    })
+    .then((data) => {
+      console.log(data)
+      setProcessedVideo(data);
+    })
+    .catch((e) => {
+      console.error('Failed to process uploaded video:', e);
```

Comment:
This is not an uploaded video but a recording - let's make this error log more relevant